src/security.md

```diff
@@ -7,17 +7,63 @@ created with it seriously.
 This page describes how to report any vulnerabilities that you find,
 and lists best practices to minimize the risk of introducing a vulnerability.
 
+## Security philosophy
+
+Dart's security strategy is based on five key pillars:
+
+* **Identify**: Track & prioritize key security risks by identifying core assets, 
+key threats and vulnerabilities. 
+* **Detect**: Detect and identify vulnerabilities using techniques and tools like
+ vulnerability scanning,
+  static application security testing and fuzzing.
+* **Protect**: Eliminate risks by mitigating known vulnerabilities and protect 
+critical assets against source threats.
+* **Respond**: Define processes to report, triage and respond to vulnerabilities 
+or attacks.
+* **Recover**: Build capabilities to contain and recover from an incident with 
+minimal impact.
+
 ## Reporting vulnerabilities
 
 To report a security issue, use [https://g.co/vulnz][].
 Coordination and disclosure happen in the [dart-lang GitHub repos][repos]
 (including [GitHub security advisories][]).
+Please include proper description of the issue,
+the steps you took to create the issue, affected versions, and any 
+mitigations for the issue.
 The Google Security Team will respond within 5 working days of
 your report on g.co/vulnz.
 
 For more information about how Google handles security issues, see
 [Google’s security philosophy][].
 
+##  Flagging existing issues as security-related
+If you believe that an existing issue is security-related, 
+we ask that you report it via [https://g.co/vulnz][] and include 
+the issue id in your report.
+
+## Supported versions
+
+We commit to publishing security updates for the version of Dart 
+currently on the `stable` branch.
+
+## Expectations
+
+We treat security issues equivalent to a P0 priority level and release a beta or
+hotfix for any major security issues found in the most recent stable version of 
+our SDK. 
```

Comment:
- This includes @mit-mit's suggestion of "patch release" 
- Changes "stable version" -> "stable release" to match other mentions on the page
- Changes "our SDK" to "Dart SDK"

```suggestion
We treat security issues equivalent to a P0 priority level 
and release a beta or patch fix
for any major security issues found
in the most recent stable release of the Dart SDK.
```